app_allocator/classes/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
flake8 error:
```
$ make code-check
classes/__init__.py:1:1: W391 blank line at end of file
make: *** [code-check] Error 1
```

Reply:
✅

Reply:
I was able to get rid of it and pushed the change.  code-check is happy in my local.